terraform/scripts/modify_variable.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#script logging to modify_variable.log file
+test x$1 = x$'\x00' && shift || { set -o pipefail ; ( exec 2>&1 ; $0 $'\x00' "$@" ) | tee -a modify_variable.log ; exit $? ; }
```

Comment:
why not just

```
exec >>modify_variable.log 2>&1
set -x
...
...

```

Reply:
this won't print output to both stdout and file in the same time if i'm not mistaken